src/Enum/LoggerChannel.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Morebec\YDB\Enum;
+
+use Morebec\ValueObjects\BasicEnum;
+
+/**
+ * LoggerChannel
+ */
+class LoggerChannel extends BasicEnum
```

Comment:
Good!

I have done some new devs on the branch to reintroduce queries, it would be great to be able to log to a query channel: 
```php
const QUERY = 'query'
```

Good on the __DEFAULT keyword circumvention.